apps/settings/style/downloads.css

```diff
@@ -235,6 +235,11 @@
 }
 
 /* RTL rules */
+html:-moz-dir(rtl) #downloadList aside.download-status {
+  right: 0.2rem;
```

Comment:
That's overridden in line 192 `#downloadList ul > li[data-state="failed"] .download-status`. So I let it to 0.2rem for now.